ee/server/calendar/google_calendar.go

```diff
@@ -162,7 +190,7 @@ func (c *GoogleCalendar) GetAndUpdateEvent(event *fleet.CalendarEvent, genBodyFn
 		return nil, false, ctxerr.Wrap(c.config.Context, err, "retrieving Google calendar event")
 	}
 	if !deleted && gEvent.Status != "cancelled" {
-		if details.ETag == gEvent.Etag {
+		if details.ETag != "" && details.ETag == gEvent.Etag {
```

Comment:
Under what circumstances does the caller provides an event with empty `details.ETag`?

Reply:
Should not happen in normal operation with Google Calendar. It is here just in case, and for testing -- we do not want to hit "" == "" and consider that the event did not update.